app/bundles/PageBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -547,7 +550,8 @@ public function onBuildJsForTrackingEvent(BuildJsEvent $event)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
   ga('create', '{$id}', 'auto');
   ga('send', 'pageview');
-  ga('set', 'userId', {$leadId});
+  {$gaUserId}
+  {$gaAnonymizeIp}
```

Comment:
The IP anonymization parameter looks like this: _&aip=1_  It is not being sent to GA on the a hit of the LP.  Move `{$gaAnonymizeIp}`  before `ga('send', 'pageview');` so the parameter is added.

Reply:
You're right. Moved set before event. Thanks